### ISSUE TITLE
Form Select "size" prop conflict with native size attr

### ISSUE THREAD

**Issue by r4fx** [bootstrap-vue/bootstrap-vue#761]
In `b-form-select` there is no possibility to use native HTML5 `size` attribute, because of prop with the same name, so please consider to change the name for `size` property to something different to enable use of `size` attribute. Thanks!

```html
<select size="3">
  <option value="volvo">Volvo</option>
  <option value="saab">Saab</option>
  <option value="opel">Opel</option>
  <option value="audi">Audi</option>
</select>
```

See how native attr `size` work
https://www.w3schools.com/Tags/tryit.asp?filename=tryhtml_select_size

**Comment by tmorehouse** [comment]
Currently `b-form-select` does not support the `multiple` attribute (neither does Bootstrap V4's custom select) which allows you to take advantage of the `size` attribute.

There is a PR #731 that introduces multiple select support, and also introduces the `multiple-size` property, which will set the `size` attribute on the select (and also removes custom styling from the select).

This PR has been merged. and will be available in v0.19.0 which should be released in the near future.

**Comment by tmorehouse** [comment]
As a side note, the prop `size` is used by many of the BootstrapVue components to control the Bootstrap V4 form-control/button sizes.

**Comment by r4fx** [comment]
yeah I seen this PR, but as you mentioned this will cover only dropdown with multiple mode,
there will not be an option to use `size` attribute in single mode. This is maybe a rare case to use it (as in my case) but i think there should be a possibility to use such native attr.

**Comment by tmorehouse** [comment]
New merged PR #762 should address the issue to allow `select-size` to be set when not in `multiple` mode.

**Comment by Sobient** [comment]
Avoid using W3Schools as a reference.

Here is a proper documentation: https://developer.mozilla.org/en/docs/Web/HTML/Element/select
